config/routes.rb

```diff
@@ -14,9 +14,12 @@
     put '/repositories/:id' => 'repositories#update', as: :repository_update
     # This route should be a POST to be semantically correct. But, RepositoriesController#create relies on a redirect to it which is not possible with a POST
     get '/repositories/:id/process' => 'repositories#process_repository', as: :repository_process
-
     get '/repository_branches' => 'repositories#branches', as: :repository_branches
 
+    scope :format => false, :constraints => { :format => 'json' } do
```

Comment:
Do we need to set the format to false if right after it we define it as to accept only json?